gems/activerecord/6.1/activerecord-6.1.rbs

```diff
@@ -31,4 +31,10 @@ module ActiveRecord
   class MismatchedForeignKey < StatementInvalid
     def initialize: (?primary_key_column: untyped? primary_key_column, ?primary_key: untyped? primary_key, ?target_table: untyped? target_table, ?foreign_key: untyped? foreign_key, ?table: untyped? table, ?binds: untyped? binds, ?sql: untyped? sql, ?message: untyped? message) -> untyped
   end
+
+  module QueryMethods
+    class WhereChain
+      def missing: (*Symbol associations) -> Relation
```

Comment:
This `Relation` is `::ActiveRecord::Relation`.
So, the model information is **missing**.
Aren't you trying to get the model information from here?

How about like this?

```rbs
module QueryMethods
  class WhereChain[Relation]
    def missing: (*Symbol associations) -> Relation
  end
end
```

Reply:
I will try it.